SPECS/helm/helm.spec

```diff
@@ -1,13 +1,13 @@
 %global debug_package %{nil}
 
 Name:          helm
-Version:       3.15.2
-Release:       3%{?dist}
+Version:       3.18.3
+Release:       1%{?dist}
 Summary:       The Kubernetes Package Manager
 Group:         Applications/Networking
 License:       Apache 2.0
-Vendor:        Microsoft Corporation
-Distribution:  Azure Linux
+Vendor:        Intel Corporation
```

Comment:
We don't normally change the vendor/distributino when we are updating a SPEC file @anujm1?

Reply:
we change Vendor and Distribution when we apply our changes to spec file ( version change or  config/packages change or patches)